### ISSUE TITLE
DNS for Services and Pods - incorrect Pod domain name

### ISSUE THREAD

**Issue by peterholak** [kubernetes/website#31310]
**DNS for Services and Pods - incorrect Pod domain name**

**Problem:**

https://github.com/kubernetes/website/blob/6833ee99e3074b43776b8ab3e26e24a952092cdd/content/en/docs/concepts/services-networking/dns-pod-service.md?plain=1#L112

shows the format

```
pod-ip-address.deployment-name.my-namespace.svc.cluster-domain.example
```

Instead of `deployment-name` (presumably the name of the mentioned Deployment / DaemonSet), it should be the name of the Service, so it should say `service-name`.

Shouldn't even matter whether the pods were created by a Deployment or not, so the text above it isn't ideal - I guess it's ok to mention that, as it is a common use case, but it could be phrased better.

Checked with nslookup inside pods on Kubernetes 1.22 to make sure.

**Proposed Solution:**

Change to

```
pod-ip-address.service-name.my-namespace.svc.cluster-domain.example
```

**Page to Update:**
https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/

**Comment by yashodhanmohan** [comment]
Hello, I've made the correcting changes after verifying that this in fact _is_ an issue in the documentation. Any pod that is selected by a service can be targeted using the DNS record and the DNS record is supposed to contain the service name instead of the DNS name. The changes are present in this [PR](https://github.com/kubernetes/website/pull/31363).

I tried looking for the actual source code in k/k where the DNS record for the pod is actually created but was unsuccessful. But someone from SIG-network would be able to review this.

/assign yashodhanmohan
